prisma/seed.ts

```diff
@@ -1,39 +1,62 @@
 import { PrismaClient } from "@prisma/client"
-import bcrypt from "bcrypt"
+import { v4 as uuidv4 } from "uuid"
 
 const prisma = new PrismaClient()
 
-async function hash(password: string) {
-  const salt = await bcrypt.genSalt()
-  return await bcrypt.hash(password, salt)
-}
-
 async function userMigrations() {
-  await prisma.user.create({
+  await prisma.account.create({
     data: {
-      email: "user@test.test",
-      username: "User",
-      role: "USER",
-      passwordHash: await hash("user"),
-      verified: true,
+      /* password: password */
+      password: "d5e0f1c480664b5cebf349bb252b2d95:d8b59da61839c29ff838df0df6b00082c21f8c999de07467b71cd19fc2dedd519042d6d0b6b043d0ac3367dec911c9ba54fd7acb4e6758159a034d4b2f713c40",
```

Comment:
What is this for a password? Is it no longer possible to just call a hash password function, maybe not bcrypt but something from betterAuth